target/linux/ath79/image/generic.mk

```diff
@@ -616,3 +624,21 @@ define Device/yuncore_a770
   IMAGE_SIZE := 16000k
 endef
 TARGET_DEVICES += yuncore_a770
+
+define Device/yuncore_cpe830
+  ATH_SOC := qca9531
+  DEVICE_VENDOR := YunCore
+  DEVICE_MODEL := CPE830
+  DEVICE_PACKAGES := uboot-envtools kmod-ath9k kmod-i2c-core kmod-i2c-gpio
```

Comment:
- `kmod-ath9k` is a target package, so no need to define it here. Same goes for `uboot-envtools`.
 - `kmod-i2c-core` is a implicit dependency of `kmod-i2c-gpio`, so it can be dropped

Reply:
done